index.js

```diff
@@ -85,6 +85,19 @@ Mailgen.prototype.generatePlaintext = function (params) {
     // Render the plaintext theme with ejs, injecting the data accordingly
     var output = ejs.render(this.cachedPlaintextTheme, ejsParams);
 
+    // Definition of the <br /> tag as a regex pattern
```

Comment:
Ooh, just thought of one more thing. We should move this entire logic to `Mailgen.prototype.cacheThemes` right after `this.cachedPlaintextTheme` because it would be better to cache this regex call instead of running it every time (which will be slow when people use Mailgen to generate 1000+ emails at a time). 👍